src/components/Banner/README.md

```diff
@@ -527,3 +527,64 @@ Banners inside of cards render with less spacing and a pared-back design to fit
 
 - To inform merchants about a new feature or opportunity, [use callout cards](/components/structure/callout-card)
 - To group similar concepts together in the interface, [use a card](/components/structure/card)
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
+
+See Material Design and development documentation about accessibility for Android:
+
+- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
+- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+Banners provide context and assist workflows for merchants with disabilities.
+
+- Critical and warning banners have a `role=”alert”`, which is announced by assistive technologies when the banner appears.
```

Comment:
Minor edit here and for the bullet below, so that they're parallel: 

```suggestion
- Critical and warning banners have a `role=”alert”` and are announced by assistive technologies when they appear.
```